crates/query-engine/sql/src/sql/convert.rs

```diff
@@ -282,6 +293,14 @@ impl From {
                 column.to_sql(sql);
                 sql.append_syntax(")");
             }
+            From::GenerateSeries { from, to } => {
+                sql.append_syntax("generate_series");
+                sql.append_syntax("(");
+                sql.append_syntax(&from.to_string());
```

Comment:
I would really like to see `.append_usize` or something; we do this a lot and it feels error-prone because changing the underlying type will not cause a compilation failure.

Reply:
Makes sense. I'll have a look at this in a separate PR.